packages/blade/src/tokens/global/sizes.ts

```diff
@@ -0,0 +1,79 @@
+export type Sizes = Readonly<{
+  /** 0: 0px */
+  0: 0;
+  /** 25: 1px */
+  25: 1;
+  /** 50: 2px */
+  50: 2;
+  /** 75: 3px */
+  75: 3;
+  /** 100: 4px */
+  100: 4;
+  /** 125: 5px */
+  125: 5;
+  /** 150: 6px */
+  150: 6;
+  /** 200: 8px */
+  200: 8;
+  /** 250: 10px */
+  250: 10;
+  /** 300: 12px */
+  300: 12;
+  /** 400: 16px */
+  400: 16;
+  /** 450: 18px */
+  450: 18;
+  /** 500: 20px */
+  500: 20;
+  /** 600: 24px */
+  600: 24;
+  /** 700: 28px */
+  700: 28;
+  /** 800: 32px */
+  800: 32;
+  /** 900: 36px */
+  900: 36;
+  /** 1000: 40px */
+  1000: 40;
+  /** 1200: 48px */
+  1200: 48;
+  /** 2000: 100px */
+  2000: 100;
+  /** 2050: 120px */
+  2050: 120;
+  /** 4000: 300px */
+  4000: 300;
```

Comment:
@divyanshu013 Where is `300` used?

Reply:
Only at [one place](https://github.com/razorpay/blade/pull/1019/files#diff-8ef63ee40c1047f4d5138dd1288573ae858afdbcd5b8bc44b6aa8d3d97a82dc0R8), for list wrapper max height.

Reply:
Got it! Good to go it seems.